x-pack/platform/plugins/shared/onechat/public/application/hooks/use_conversation.ts

```diff
@@ -32,7 +36,7 @@ const useConversation = () => {
     },
   });
 
-  return { conversation, isLoading };
+  return { conversation, isLoading, isFetched };
```

Comment:
Let's chat about the best way to expose these kind of properties to UI components @zacharyparikh -- this solution works now, but, I wouldn't necessarily advocate for this. We can come up with a better pattern together that suits the architecture of the codebase.